Add example files for setup

domaine-preston opened the pull request:
Creating example `npmrc` and `env` files for assisting with project setup

treyhardin reviewed (changes requested):
.env file looks great! I'm not sure we need the .npmrc though, since we can just commit it to the repo. Lmk if you think there's a case to leave this in!

domaine-preston commented:
> .env file looks great! I'm not sure we need the .npmrc though, since we can just commit it to the repo. Lmk if you think there's a case to leave this in!

hey @treyhardin yep you're right. I got confused. I removed it and added the note about the error to the actual `.npmrc` file.
